src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,92 @@
  */
 
 public class Plane {
+    private int id;
+    private String name;
+    private int crewCount;
+    private String producer;
+    private double height;
+    private double length;
+    private double wingspan;
+    private double weight;
 
+    public Plane(PlaneBuilder planeBuilder) {
```

Comment:
Should this class be public?

Reply:
Sorry i forgot that we don't invoke that constructor in Main class. Thanks for advice

Reply:
alredy fixed that